### ISSUE TITLE
(css) not all valid attributes are supported (svg related)

### ISSUE THREAD

**Issue by velara3** [highlightjs/highlight.js#3606]
**Describe the issue**
The property names in a block of CSS do not appear to get highlighting

**Which language seems to have the issue?**
CSS. The container has language 

**Are you using `highlight` or `highlightAuto`?**
 I am using hljs.hightlight();

`var cssValue = hljs.highlight(cssOutput, {language: 'css'}).value;`

**Sample Code to Reproduce**

// does not highlight any of the property names in the id selector block //

```
#Rectangle_1_p {
	fill: rgba(255,101,101,1);
	stroke: rgba(112,112,112,1);
	stroke-width: 1px;
	stroke-linejoin: miter;
	stroke-linecap: butt;
	stroke-miterlimit: 4;
	shape-rendering: auto;
}
.Rectangle_1_p {
	position: absolute;
	overflow: visible;
	width: 212px;
	height: 119px;
	left: 26px;
	top: 47px;
}
```
The property names in the class selector are highlighted but the property names in the ID selector are not. 

**Expected behavior**
Property names to be highlighted

Screenshot: 
<img width="225" alt="image" src="https://user-images.githubusercontent.com/1084127/187515702-001f99f8-a253-4261-96e0-c7bf25d2b6c4.png">

**Comment by joshgoebel** [comment]
Related: #3484

Nothing to do with the selectors, trying putting all the rules inside a simple `body` rule, the problem is our list of CSS properties simply isn't exhaustive (and some question of whether it should be).

I thought we had an open issue on this (gaps in CSS support), but I can't find it now.  

I guess since CSS (all alone) is only 3.8kb (minified) that we technically have room for more here...   It's looks like that seems to be kind of the problem area...  Should largely be a matter of updating the array of attributes or perhaps creating a new SVG attributes array (if they are so special) and then merging them at runtime...

Would you be willing to work on a PR that adds SVG related attributes?

**Comment by velara3** [comment]
I can take a look at it. Can you direct me to the file to be modified?

**Comment by joshgoebel** [comment]
Look at css and css-shared.  You might want to read our build docs and find our developer tool in /tools/

**Comment by Denyweeeed** [comment]
...

**Comment by MohamedAli00949** [comment]
Hi there, I have read the code base and can work on this issue, please assign it to me.

**Comment by joshgoebel** [comment]
We don't really assign issues, but feel free to submit a PR and we're happy to have a look.

**Comment by joshgoebel** [comment]
Is this closed by #3893?  If not, please let us know.
